index.md

```diff
@@ -1,15 +1,55 @@
 ---
 layout: default
-title: Why Data Commons
-nav_order: 1
+title: What is Data Commons?
+nav_order: 0
 ---
 
-# Why Data Commons?
+{: .no_toc}
+# What is Data Commons?
```

Comment:
Should we point to the About page at all, for the Why?

Reply:
I feel like, if they've made it this far, they've probably read the About stuff already and we probably don't need a circular link back. WDYT?

Reply:
ok, that makes sense too.

perhaps we could add a link to the paper at the end of this page, for the "why" (and also "how")

Reply:
That's a nice idea. Added.